extension/intents/routines/routineExecutor.js

```diff
@@ -85,14 +88,19 @@ export class RoutineExecutor {
       exc.displayMessage = "Another routine is already active.";
       throw exc;
     }
+    currentRoutineExecutor = this;
     for (
       ;
       this.programCounter < this.subcommands.length;
       this.programCounter++
     ) {
+      if (this.stopRoutine === true) {
```

Comment:
Unless `stopRoutine` can be something besides true/false, then it would be best to just say `if (this.stopRoutine)`